src/connectionController.ts

```diff
@@ -713,6 +713,43 @@ export default class ConnectionController {
       : '';
   }
 
+  _getConnectionStringWithProxy(mongoClientConnectionOptions: { url: string; options: MongoClientOptions; }): string {
```

Comment:
Should we also use this method when we `Copy connection string`?
https://github.com/mongodb-js/vscode/blob/main/src/connectionController.ts#L725

Reply:
We can copy a connection string for any connection on a sidebar, not only the active one. But proxy options are available only for the current connection, therefore we don't change the copy connection string behaviour.